### PR TITLE
Update logrotate.c

### PR DESCRIPTION
**Pull request by glensc**
typo fix

### CODE REVIEW

File: logrotate.c [hunk 1]
```diff
@@ -1137,7 +1137,7 @@ int findNeedRotating(struct logInfo *log, int logNum, int force)
 	if (log->minsize && sb.st_size < log->minsize) {
 	    state->doRotate = 0;
 	    message(MESS_DEBUG, "  log does not need rotating "
-		    "('misinze' directive is used and the log "
+		    "('minsize' directive is used and the log "
 		    "size is smaller than the minsize value");
```
Comment (sergey-v) [inline logrotate.c]: Looks like the message still lacks ')' symbol in text (after 'value' word).

Comment (glensc) [inline logrotate.c]: eh. adding

Comment (sergey-v) [inline logrotate.c]: Thanks!

**Comment by kdudka** [comment]
Thanks for the patches!

**Pull request closed without merging**